# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/libdav1d/vcs_version.h
# Commit: abcd7426fb63
# Full Hash: abcd7426fb63f078b3d5e2566c28baa1dd923616
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-04-23 09:53:27
# Description:
#   Bug 1546070 - Update dav1d from upstream to f8cac8c. r=TD-Linux
#   
#   The imported version is just one commit over 0.2.2 and solves the crash for this bug.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D28343
# ==============================================================================

diff -r bb714fa13ef9 -r abcd7426fb63 media/libdav1d/vcs_version.h
--- a/media/libdav1d/vcs_version.h	Mon Apr 22 20:41:38 2019 +0000
+++ b/media/libdav1d/vcs_version.h	Mon Apr 22 19:48:05 2019 +0000
@@ -1,2 +1,2 @@
 /* auto-generated, do not edit */
-#define DAV1D_VERSION "0.2.2-0-g1f7a7e8"
+#define DAV1D_VERSION "0.2.2-1-gf8cac8c"